app/models/filter.rb

```diff
@@ -135,6 +141,21 @@ def expire_topbar_cache(sweeper)
     role.usergroups.each { |g| g.expire_topbar_cache(sweeper) }
   end
 
+  def disable_overriding!
+    self.override = false
+    self.save!
```

Comment:
There's something I don't understand about Rails, why does `inherit_taxonomies!` persist without calling save, but this one does not? 

After removing the `.save!` call
```

irb(main):012:0> filter.update_attributes(:override => true)
=> true
irb(main):013:0> filter.override
=> true
irb(main):014:0> filter.disable_overriding!
=> false
irb(main):015:0> filter.override
=> false
irb(main):016:0> filter.reload
irb(main):016:0> filter.override
=> true
```

Nothing against making the model save itself, but I don't understand why the other method doesn't need it :disappointed: